src/OrleansTestingHost/Extensions/TestConfigurationExtensions.cs

```diff
@@ -14,37 +14,40 @@ public static class TestConfigurationExtensions
         /// <summary>
         /// This call tweaks the cluster config with settings specific to a test run.
         /// </summary>
-        public static void AdjustForTestEnvironment(this ClusterConfiguration clusterConfig)
+        public static void AdjustForTestEnvironment(this ClusterConfiguration clusterConfig, string dataConnectionStringFallback)
```

Comment:
well, this IS that code you are talking about, the one responsible to setting the connection string in the cluster config. I'll play with it a little bit to see how I can improve it further